cppcache/include/geode/PoolFactory.hpp

```diff
@@ -544,6 +545,7 @@ class _GEODE_EXPORT PoolFactory {
 
   friend class Cache;
   friend class PoolManager;
+  friend class PoolManagerImpl;
```

Comment:
Why this friend?

Reply:
`PoolManagerImpl::createFactory` calls into `PoolFactory` constructor and `PoolFactory` constructor is private.  I've wondered is if these constructors even need to be private anymore?  I assumed it used to be to protect the original singleton design.  Perhaps this should be a separate ticket to investigate.

Reply:
Ok. It is more likely private to require the use of the factory method. Just odd to expose internal classes in a public header like this. Should be ok for now.